### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Rustamxon7**
**I have completed all the requirements:**

- Deleted all unused files. ✅
- Created components directory. ✅
- Calculator.js created. ✅
- Used class-based components. ✅
- Tested on browser if it is working or not. ✅

Feel free to review -🧑🏻‍💻- my code and let me know if there are any changes to do.-🤓-😉-

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
Comment (EricMbouwe) [inline src/components/Calculator.js:7]: - [ ] As mentioned above you don't need a state here 😉. Let's remove it for now 🙂.

File: src/index.css [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+  font-size: 2rem;
+  color: #fff;
+  padding: 2rem 1rem;
+}
+
+.keypad {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-auto-rows: minmax(7rem, auto);
+}
```
Comment (EricMbouwe) [inline src/index.css:38]: - [ ] The keypad on your calculator does not match the given design as required for this project. The options buttons don't have the appropriate background color. I have noticed that you added a `className="options"` to them, please go ahead and add that background color to that selector.

Screenshot from 2021-09-21 22-02-33 
Screenshot from 2021-09-21 22-12-47

File: src/App.js [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```
Comment (EricMbouwe) [inline src/App.js:8]: - [ ] Humm... At this point, you don't need a state here 😉. You can double-check the requirements `Develop just the UI of the calculator; the math logic to make it work will be added in the next project.` Let's remove it for now 🙂.

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @Rustamxon7, 🙋‍♂️

Good job so far!

- Nice looking readme 👍
- Descriptive commit messages 👍
- Well written PR 👍

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Rustamxon7, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**